src/agent/service.ts

```diff
@@ -51,7 +51,8 @@ if (typeof svcinstall[action] !== 'function') {
 }*/
 
 // node is known as nodejs on some *nix installs
-var nodePath = shelljs.which('nodejs') || shelljs.which('node');
+//var nodePath = shelljs.which('nodejs') || shelljs.which('node');
+var nodePath = path.resolve('__dirname/..', 'runtime/node/bin/node')
```

Comment:
did this work?  typically __dirname is not quoted - but maybe it works.

Typically, it would be written as:
path.join(__dirname, '..', 'runtime/node/bin/node')

Only need to use resolve if the inputs can be relative or absolute,  In this case, it's relative so resolve is not needed.

We should cherry-pick to releases/0.4 as well